_posts/en/newsletters/2019-01-22-newsletter.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Bitcoin Optech Newsletter #30'
+permalink: /en/newsletters/2019/01/22/
+name: 2019-01-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed LN feature to allow making
+spontaneous payments and provides our longest-ever list of notable code
+changes to popular Bitcoin infrastructure projects.
+
+## News
+
+- **PR opened for spontaneous LN payments:** LN protocol developer
+  Olaoluwa Osuntokun opened a [pull request][spontaneous payments] to
+  allow an LN node to send a payment to another node without first
+  receiving an invoice.  This takes advantage of LN's Tor-like onion
+  routing by allowing a spender to choose a preimage, encrypt it so
+  that only the receiver's node can decrypt it, and then route a payment
+  along LN like normal using the the hash of the preimage.  When the
+  payment reaches the receiver, they decrypt the preimage and disclose
+  it to the routing nodes in order to claim the payment.
+
+    Allowing spontaneous payments allows nodes to accept donations
```

Comment:
You've led with donations here, and then also mentioned 'ad hoc payment tracking'. I'm not sure if donations will be predominant use case here. You could consider starting with a more general statement like 'Spontaneous payments may open up new use cases where invoices and proof of payment are not required. For example...'